src/__tests__/database/server/boardMember.test.ts

```diff
@@ -119,24 +119,23 @@ describe('patchBoardMember()', () => {
     it('Throws for user error', async () => {
         const boardId = 1451;
         await expect(patchBoardMember(boardId, { })).rejects.toThrow('user input error');
-        expect(mockQuery).toHaveBeenCalled();
```

Comment:
This test was supposed to cover the 1451 case in the error handling, so I'd suggest adding a request body instead of removing the mock query assertion. There should also be another test that's testing for the empty request like you're doing here.

Reply:
Good catch, In the new commit I added a test for the 1451 case in `patchBoardMember.`

I also added a test to catch the empty request user error in `insertBoardMember` as well